src/controllers/itemDetails/index.html

```diff
@@ -110,8 +108,6 @@
                         <div class="recordingFields hide" style="margin: 0.5em 0 1.5em;"></div>
 
                         <div class="detailSectionContent">
-                            <div class="itemLastPlayed hide"></div>
```

Comment:
Introduced in https://github.com/jellyfin/jellyfin-web/pull/459
Removed in https://github.com/jellyfin/jellyfin-web/pull/949